protected/humhub/components/behaviors/PolymorphicRelation.php

```diff
@@ -129,14 +132,23 @@ public function resetPolymorphicRelation()
         $this->cached = null;
     }
 
+    /**
+     * Returns if the polymorphic relation is established
+     * @since 1.16
+     */
+    public function isPolymorphicRelation(): bool
```

Comment:
```suggestion
    public function isPolymorphicRelationLoaded(): bool
```

For me the name is not quite clear.

Reply:
Cool.